src/ast/passes/codegen_llvm.cpp

```diff
@@ -1706,7 +1721,13 @@ void CodegenLLVM::binop_int(Binop &binop)
     default:
       LOG(BUG) << "\"" << opstr(binop) << "\" was handled earlier";
   }
-  // Using signed extension will result in -1 which will likely confuse users
+
+  return;
+
+sign_extend_bool:
+  // Just the bottom bit needs to be set for bools. If we do a signed sign
```

Comment:
a signed sign?

Reply:
Fixed other typo too. And tightened up the language